app/docs/docs/testing/e2e-testing.md

```diff
@@ -0,0 +1,93 @@
+# E2E Testing
+
+## Pattern
+
+For our e2e tests we are using a custom version of the [POM](https://playwright.dev/docs/pom) (Page Object Model). The idea is for the class to handle all the locators and expects, abstracting each step of the testing to make the tests themselves clear and easy to follow. We also want to design the tests in a user flow centric way, putting focus on ensuring a user can perform a given action without encountering issues along the way.
```

Comment:
I'd say custom version is a bit misleading -- I'd just put POM. Also, typically you used the word first then put in parentheses the acronym. E.g. Page Object Model (POM)